### ISSUE TITLE
Scrobbling to anime tracking websites

### ISSUE THREAD

**Issue by dulleh** [dulleh/akhyou#13]
Scrobbling (updating watched episodes automatically) to anime tracking websites such as HB and MAL. Completely optional of course. The account would be displayed in the sidebar.

**Comment by dulleh** [comment]
[Hyped for no 1 enhancement.](https://github.com/dulleh/akhyou/issues/1#issuecomment-145434430) - @Syborg49

**Comment by Syborg49** [comment]
Yes. This would be phenomenal. Keeping track of what I've watched. Your app is almost perfect.

**Comment by dulleh** [comment]
Oh, why thank you. This is an important feature to me too. I'm busy with school currently but soon as I'm done I'll probably get to work on this. (maybe get MAL up first just cos it's probably easier to work with).

**Comment by Syborg49** [comment]
That's awesome! No worries, school comes first, is MAL that much easier to code for than HM?

**Comment by dulleh** [comment]
I'm actually not sure. I just know that the HB api for logging in is changing and so that's why I've been putting off building this feature. There's also the fact that the HB search is balls (or it was last time I checked) and so it's probably easier to match anime from random streaming sites to their equivalent entries on MAL. There's a lot I still have to think about with the implementation of this however, so it may not necessarily be the case that one is harder than the other.

**Comment by wopian** [comment]
The MAL "API" is a nightmare.

**Comment by dulleh** [comment]
Well there go all my hopes and dreams. Is the HB one any better?

**Comment by wopian** [comment]
It's pretty [straightforward](https://github.com/hummingbird-me/hummingbird/wiki/API-v1-Methods#library--addupdate-entry). I highly doubt OAuth will come out before V3 (and API V3).

**Comment by dulleh** [comment]
That sounds good @wopian. Thanks. Looks like this feature will come faster than expected.

**Comment by dulleh** [comment]
"faster than expected"... I have got down to seriously working on this in the past couple weeks but there's still quite a bit of work to do. Even so, putting this on hold for the next couple weeks as I have mocks.

**Comment by dulleh** [comment]
Functionality for having a user saved and displayed in the sidebar was added in 7d22de5a7a9c8ad32cf954142fd0aa306aa9c2fe

**Comment by dulleh** [comment]
Wow, a whole year and still not done. Kitsu api here http://docs.kitsu17.apiary.io/#

**Comment by Lionirdeadman** [comment]
You could do Anilist, they have an API. Here's the docs : http://anilist-api.readthedocs.io/en/latest/.
They also have a discord server if you ever need help or just want to talk : https://discord.me/anilist
Thanks for the work you're doing.
